src/components/Alert/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import errorIcon from 'assets/images/icons/error.svg'
+
+type AlertProps = {
+  title: string
+  detail: string
+}
+
+const Alert = ({ title, detail }: AlertProps): JSX.Element => {
+  return (
+    <div className="alert">
+      <img src={errorIcon} alt="error" />
+      <div className="alert-message">
```

Comment:
As per the rule we follow this element class should be named as `alert__message`, for title `alert__message-title`.
![image](https://user-images.githubusercontent.com/14927672/122162224-038db680-ce95-11eb-95c0-6998317211a3.png)

What do you think?  🤔

Reply:
Agree with you. Fixed in 1243d220a611cdeac071166ba804331ed3aa5ab6